pkg/ttnpb/udp/translation.go

```diff
@@ -411,7 +409,6 @@ func FromDownlinkMessage(msg *ttnpb.DownlinkMessage) (*TxPacket, error) {
 	tx.DatR.DataRate = scheduled.DataRate
 	switch scheduled.DataRate.GetModulation().(type) {
 	case *ttnpb.DataRate_Lora:
```

Comment:
This is a conversion function to the UDP format. Do not remove the `tx.CodR` assignment, but instead use the same `mod := ...` in the switch and fill it with `mod.Lora.CodingRate`.